README.md

```diff
@@ -155,6 +156,70 @@ StatusCode = {
 };
 ```
 
+## Support of IndexedDB
+This library provides the support of storing private keys in the browser [indexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API).
+The function can be enabled by setting `scalar.dl.client.private_key_indexeddb_enabled` to `true` in the client properties.
+
+As indexedDB function is enabled, depending on whether a private key is specified in the properties,
```

Comment:
I think we should simply explain what we should do when using IndexedDB.
I think we do the following steps in most of cases, do we ?

1. Specify `enabled: true` without `private_key_pem` specified.
2. If 1 throws error, get a private key from somewhere and specify `enabled: true` with `private_key_pem` specified with the private key.
3. If 1 doesn't throw error, go on with the key stored key in IndexedDB.